### ISSUE TITLE
Dataset on the applicability of this method

### ISSUE THREAD

**Issue by SuzyZhao0417** [ShunyuanZheng/GPS-Gaussian#25]
Hello！ Thanks for your excellent work! But I have some question about the dataset. Does the method only apply to humans dealing with ring scenes and can it be used to deal with LLFF data？As it is mentioned in the paper that the purposed method need to select the left and right cameras as source views and relies on the results of binocular depth estimation. Looking forward to your reply! Best wishes!

**Comment by ShunyuanZheng** [comment]
Hi @SuzyZhao0417 , currently our method mainly deals with the human-centered scene. Our method needs GT depth for training and it is not straightforward to synthesize training data with GT depth of background regions. We have tried to train GPS-Gaussian without depth supervision which enables the application on more general scenarios (with background) such as LLFF and ENeRF outdoor dataset. By making some modifications and attaching additional regularization terms, it works. Thanks to the fully differentiable framework, our method can achieve a competitive rendering performance even with a degenerate geometry proxy rebuilt from an unsupervised depth estimation. However, this degrades the quality of the live demo, and I think more efforts should be paid to improve the performance.

**Comment by SuzyZhao0417** [comment]
If I want to test on LLFF data what format do I need to process the data？

**Comment by arnabdeypolimi** [comment]
@ShunyuanZheng did you understand the format of the extrinsic params? W2C?

**Comment by ShunyuanZheng** [comment]
> @ShunyuanZheng did you understand the format of the extrinsic params? W2C?

Yes, the provide extrinsic is in world-to-camera format in OpenCV coordinate.
